Singapore/src/main/java/com/infoshareacademy/model/Place.java

```diff
@@ -0,0 +1,20 @@
+package com.infoshareacademy.model;
+
+import org.json.simple.JSONObject;
+
+public class Place extends PersistentAbstract{
+    private String name;
+    private String description;
+    private double prize;
+    private double rate;
+    private String opinion;
```

Comment:
Aby opinie były generyczne należy je dodać jako obiekt

```suggestion
    private Opinion opinion;
```

Reply:
To zrobi Krzysiek